docker-compose.example.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+services:
+  daemon:
+    build: .
+    restart: always
+    hostname: daemon
+    ports:
+      - "8080:8080"
+      - "2022:2022"
+    tty: true
+    environment:
+      - "DEBUG=false"
+    volumes:
+      - "/var/run/docker.sock:/var/run/docker.sock"
+      - "/var/lib/docker/containers/:/var/lib/docker/containers/"
+      - "/var/lib/pterodactyl/:/var/lib/pterodactyl/"
+      - "/srv/daemon-data/:/srv/daemon-data/"
+      - "/tmp/pterodactyl/:/tmp/pterodactyl/"
```

Comment:
Do we even need the temp files outside the container? Seems unnecessary. Same for the timezone file? I actually think that is needed now that I think about it, but I wonder if we shouldn't attempt to read the environment timezone first before trying to use the file, that would certainly make this a little less volume heavy.

Reply:
There are reasons for each and every volume.

`/var/run/docker.sock` - control docker outside the container.
`/var/lib/docker/` - read docker logs
`/var/lib/pterodactyl/` - default for pterodactyl config
`/srv/daemon-data/` - default for data storage needed for web file manager
`/tmp/pterodactyl/` - how you pass the install.sh files

Reply:
Just because I'm reading it here: We should really rename the default data location to something like `/srv/wings` instead of `/srv/daemon-data/`, which is way to generic.

Reply:
I would be for that as well honestly.